xds/adapter/rds.go

```diff
@@ -226,7 +226,7 @@ func mkEnvoyVirtualHost(
 		corsPolicy = mkEnvoyCorsPolicy(cleanDomainConfig.Domain.CorsConfig)
 	}
 
-	domains := []string{cleanDomainConfig.Domain.Name}
+	domains := []string{cleanDomainConfig.Domain.Addr()}
```

Comment:
Can you talk about why you made this change, and what problem you're fixing? it's not clear from your review description.

Reply:
I need to also add the `name:port` for `tcp`/`grpc` connections which require `:port`.

Reply:
I'm fine with adding both.